c++/include/orc/Writer.hh

```diff
@@ -266,17 +266,15 @@ namespace orc {
     bool getUseTightNumericVector() const;
 
     /**
-     * Set the initial capacity of output buffer in the class BufferedOutputStream.
-     * Each column contains one or more BufferOutputStream depending on its type,
-     * and these buffers will automatically expand when more memory is required.
+     * Set the initial block size of input buffer in the class CompressionStream.
      */
-    WriterOptions& setOutputBufferCapacity(uint64_t capacity);
+    WriterOptions& setMemoryBlockSize(uint64_t capacity);
```

Comment:
We cannot remove or rename any public API. This will immediately break downstream code once they upgrade to this version. If we think these APIs are not used any more, we'd better add `[[deprecated]]` attribute.